roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -14,18 +14,29 @@ confluent:
 
 jolokia_enabled: true
 jolokia_jar_path: /opt/jolokia/jolokia.jar
+zookeeper_jolokia_enabled: "{{jolokia_enabled}}"
 zookeeper_jolokia_port: 7770
+kafka_broker_jolokia_enabled: "{{jolokia_enabled}}"
 kafka_broker_jolokia_port: 7771
+schema_registry_jolokia_enabled: "{{jolokia_enabled}}"
 schema_registry_jolokia_port: 7772
+kafka_connect_jolokia_enabled: "{{jolokia_enabled}}"
 kafka_connect_jolokia_port: 7773
+ksql_jolokia_enabled: "{{jolokia_enabled}}"
 ksql_jolokia_port: 7774
+kafka_rest_jolokia_enabled: "{{jolokia_enabled}}"
 kafka_rest_jolokia_port: 7775
 
 jmxexporter_enabled: false
-jmxexporter_install_path: /opt/prometheus/
+# jmxexporter_install_path: /opt/prometheus
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
+zookeeper_jmxexporter_enabled: "{{jmxexporter_enabled}}"
+zookeeper_jmxexporter_config_path: /opt/prometheus/zookeeper.yml
+zookeeper_jmxexporter_port: 8079
```

Comment:
@thecrazymonkey I had a port collision with zookeeper and kafka running on the same host, so I set zookeeper's port to 8079... (8081 is taken by SR) . This can be set back to 8080 in your hosts.yml if necessary